ngraph/core/include/ngraph/descriptor/output.hpp

```diff
@@ -63,6 +66,10 @@ namespace ngraph
             const std::vector<Input*>& get_inputs() const { return m_inputs; }
             Tensor& get_tensor() const;
 
+            using RTMap = std::map<std::string, std::shared_ptr<Variant>>;
```

Comment:
Why do you need to add RT info to Output port?

Reply:
This is the first case when we have an attribute (Mask) that belongs to output port. Looking forward we will have more such attributes (LPT, Transpose syncing).

Reply:
I propose to discuss this topic offline.

Reply:
Discussed offline. Decided to keep rt_info in descriptor::Output to have ability to set different attributes in Input and Output ports.